Makefile

```diff
@@ -408,8 +409,9 @@ sdk-check-version:
 	go run tools/sdkver/sdkver.go --check-major=0 --check-patch=0
 
 mockgen:
-	go get github.com/golang/mock/gomock
-	go get github.com/golang/mock/mockgen
+	GO111MODULE=off go get github.com/golang/mock/gomock
+	GO111MODULE=off go get github.com/golang/mock/mockgen
+	cp -r /go/src/github.com/golang/mock ./vendor/github.com/golang/
```

Comment:
Instead of assuming a GOROOT directory, we can do something like https://github.com/libopenstorage/operator/blob/master/tools.go#L8 to make sure go mod pulls the mockgen correctly?